### PR TITLE
Refactoring Changes on DataLoss Report 

### PR DESCRIPTION
**Pull request by AyanDuttaHMCTS**
### JIRA link (if applicable) ###

PAY-2808

### Change description ###

Refactoring Changes on DataLoss Report 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -24,4 +24,10 @@
     <cve>CVE-2020-8022</cve>
     <cve>CVE-2017-18640</cve>
   </suppress>
+
+  <!--Added on 20th May 2020-->
```
Comment (YasharRahvar) [inline config/owasp/suppressions.xml:28]: Do not to add new CVEs unless there is no solution for it at the moment.

**Review by YasharRahvar — CHANGES_REQUESTED** [review]

**Review by AnilDhawaleGitHub — APPROVED** [review]

**Review by aravindan89 — APPROVED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (AyanDuttaHMCTS) [reply config/owasp/suppressions.xml:28]: Not sure about these CVEs, they exists in last change, But will see if they are valid with recent CVE fixes.

Comment (AyanDuttaHMCTS) [reply config/owasp/suppressions.xml:26]: I have checked the CVEs CVE-2017-18640, CVE-2020-8022 we still need to suppress as we are using the following Jars,
snakeyaml-1.25.jar
tomcat-embed-core-9.0.31.jar
tomcat-embed-websocket-9.0.31.jar

Comment (YasharRahvar) [reply config/owasp/suppressions.xml:28]: What I mean is to try to upgrade the dependencies and resolve the CVE error instead of suppressing them

Comment (AyanDuttaHMCTS) [reply config/owasp/suppressions.xml:28]: These tomcat & snake libs are transitive dependencies of Spring Boot
I think we upgraded Spring Boot for All the Apps.

**Review by YasharRahvar — APPROVED** [review]

**Review by krishnanuthalapati — APPROVED** [review]

**Pull request merged**